src/components/ProcessForm/ProcessForm.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { Col, Form } from "react-bootstrap";
+import { getBlankProcess, IPerson, IProcess, ParentOrgs, Person, ProcessTypes, SetAsideRecommendations } from "../../api/DomainObjects";
+import { IProcessValidation, ProcessValidation } from "../../utils/ProcessValidation";
+import { InfoTooltip } from "../InfoTooltip/InfoTooltip";
+import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import { SubmittableModal } from "../SubmittableModal/SubmittableModal";
+import "./ProcessForm.css"
+
+export interface IProcessFormProps {
+    processType: ProcessTypes,
+    showModal: boolean,
+    handleClose: () => void,
+    submit: (process: IProcess) => Promise<any>
+}
+
+export const ProcessForm: FunctionComponent<IProcessFormProps> = (props) => {
+
+    const [process, setProcess] = useState<IProcess>(getBlankProcess(props.processType));
+    const [validation, setValidation] = useState<IProcessValidation>();
+
+    useEffect(() => {
+        // Update validation whenever a field changes after a submission attempt
+        if (validation) {
+            setValidation(ProcessValidation.validateProcess(process, ["OZI", "OZJ", "OZA"]));
```

Comment:
Is the plan to pull this from a master list somewhere later?

Reply:
Yeah, I made a card for that while I was completing this one